packages/studio-base/src/components/OpenDialog/Start.tsx

```diff
@@ -73,12 +73,17 @@ export default function Start(props: IStartProps): JSX.Element {
     [theme],
   );
 
+  const supportedLocalFiles = useMemo(
+    () => Array.from(new Set(supportedFileExtensions)).join(", "),
```

Comment:
Nit: You can go even more compact using the spread operator

```suggestion
    () => […new Set(supportedFileExtensions)].join(", "),
```